core/src/hlist.rs

```diff
@@ -552,6 +552,30 @@ impl<Head, Tail> HCons<Head, Tail> {
         Selector::get(self)
     }
 
+    /// Mutably borrow an element by type from an HList.
+    ///
+    /// # Examples
+    ///
+    /// ```
+    /// # #[macro_use] extern crate frunk; fn main() {
+    /// let mut h = hlist![1i32, true];
+    ///
+    /// // type inference ensures we fetch the correct type
```

Comment:
Start sentence with capital T and end with punctuation.

Reply:
I had similar nits in #124 =P

I think we should follow standard AmE writing per [RFC 1574](https://github.com/rust-lang/rfcs/blob/master/text/1574-more-api-documentation-conventions.md) including proper punctuation :)

Reply:
Well, it says "all documentation." I don't regard comments as documentation, and I honestly think `// Won't compile` at the end of a line looks hilarious, even moreso if you follow it up with a period.

Anyways, it doesn't bother me that much. Just warning you that you'll have to remind me, because I *will* forget. :stuck_out_tongue: